tests/import_test.go

```diff
@@ -129,6 +129,58 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 	})
 })
 
+var _ = Describe("Importer respects node placement", func() {
+	var cr *cdiv1.CDI
+	var oldSpec *cdiv1.CDISpec
+	f := framework.NewFramework(namespacePrefix)
+
+	// An image that fails import
+	invalidQcowLargeSize := func() string {
+		return fmt.Sprintf(utils.InvalidQcowImagesURL+"invalid-qcow-large-size.img", f.CdiInstallNs)
+	}
+
+	BeforeEach(func() {
+		var err error
+		cr, err = f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		oldSpec = cr.Spec.DeepCopy()
+		nodes, err := f.K8sClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		cr, err := f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		cr.Spec = *oldSpec.DeepCopy()
+		_, err = f.CdiClient.CdiV1beta1().CDIs().Update(context.TODO(), cr, metav1.UpdateOptions{})
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
I would verify that the CDI has been properly restored before continuing. Its likely not a problem right now because no other tests currently look at the node placement, but if we add other tests that do look at it, this can leave the CR in a state that test is not expecting.

Reply:
Forgot to reply to this but I do check it now, it makes a second test fail if I don't :-)

Reply:
I am do something like
```go
Eventually(func() bool {
  cr, err = f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
  Expect(err).ToNot(HaveOccurred())
  return reflect.DeepEqual(cr.Spec, oldSpec)
}, 30 * time.Second, time.Second)
```
So that we verify that the CR has been restored to the original state before running the next test. As I read it right now, it is possible that the update is slow, and the next test runs and reads the incorrect CR value because the update hasn't finished yet. This particular scenario has happened in the past for other objects.

Reply:
Sorry, got confused with the other test at first, this makes sense and I added the Eventually.